prusti/tests/verify/pass/mut-borrows/refs-in-tuples-1.rs

```diff
@@ -0,0 +1,19 @@
+fn main() {
```

Comment:
Could you move these tests to `prusti/tests/pass/parse/` so that they are executed by CI?